rhc-ose-ansible/integration.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- hosts: localhost
+  vars_prompt:
```

Comment:
@JayKayy i don't think there's any reason to prompt for these. Just put them in your input inventory file, and show that they get output to the generated inventory file.